src/webserver.py

```diff
@@ -79,7 +79,7 @@ def command(self) -> list[str]:
             The command to start the Gunicorn web server.
         """
         return [
-            "python3",
+            "/bin/python3",
```

Comment:
I don't think it's a good idea, if we want to keep supporting custom OCI image for users, hardcoding the path here will cause troubles
I see 2 options:
* Appending "/bin" to the $PATH so that python3 will be found
* Providing a configuration option